### PR TITLE
Sticker message

### PR DESCRIPTION
**Pull request by godoakbrutal**

### CODE REVIEW

**Comment by liverbool** [comment]
Can you add something like this?

```php
StickerTemplate::createBrowny('stickerId');
```

**Comment by godoakbrutal** [comment]
@liverbool 
Done

File: src/Template/StickerTemplate.php [hunk 1]
```diff
@@ -23,4 +23,40 @@ public function getTemplate()
...
+
+    /**
+     * @param int $stickerId
+     */
+    public function createDaily($stickerId)
+    {
+        $this->packageId = 4;
+        $this->stickerId = $stickerId;
+    }
 }
```
Comment (liverbool) [inline src/Template/StickerTemplate.php]:
```php
public static function createDaily($stickerId)
{
    return new self(4, $stickerId);
}

Comment (godoakbrutal) [reply src/Template/StickerTemplate.php]: Done

Comment (liverbool) [reply src/Template/StickerTemplate.php]: have no construct

```php
public static function createDaily($stickerId)
{
    $self = new self();
    $self->packageId = 4;
    $self->stickerId = $stickerId;

    return $self;
}
```

**Comment by liverbool** [comment]
have no construct

```php
public static function createDaily($stickerId)
{
    $self = new self();
    $self->packageId = 4;
    $self->stickerId = $stickerId;

    return $self;
}
```

**Comment by godoakbrutal** [comment]
Ok kub

**Comment by liverbool** [comment]
add
  - copyright
  - author
  - comments

**Comment by godoakbrutal** [comment]
@liverbool 
Do we have a copyright yet ?

File: src/Message/StickerMessage.php [hunk 2]
```diff
@@ -1,10 +1,22 @@
 <?php
 
+/*
+ * This file is part of the PhpMob package.
```
Comment (liverbool) [inline src/Message/StickerMessage.php]: PhpMob?

File: src/Message/StickerMessage.php [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+ * file that was distributed with this source code.
+ */
+
 namespace LineMob\Core\Message;
 
 use LineMob\Core\Command\AbstractCommand;
 use LineMob\Core\Template\StickerTemplate;
 
+/**
+* @author Taned Suparpornhemin <taned@intbizth.com>
```
Comment (liverbool) [inline src/Message/StickerMessage.php]: dyh none office email?

**Comment by liverbool** [comment]
ps. add copyright to all files.

**Pull request merged**